PathOfBuildingCommunity/PathOfBuilding
# Export cluster jewel index data, mess with jewel rendering logic

### Description of the problem being solved:
Importing characters did not always allocate cluster jewels properly, due to nodes having an incorrect orbitIndex
### Steps taken to verify a working solution:
- Import character "TwiliteAvenger" from account "The9Who", all nodes should be allocated on the small clusters
- Import character "ArchnemesisJay" from account "Dreamiixz", the keystone should be allocated
